### PR TITLE
fix: multiple - fix LogRequest, add docs, clean

### PR DESCRIPTION
**Pull request by danbosnichill**
- fix: issue with to_underscore being called on symbol fields …
- fix: remove platform_id
- fix: remove event_api_timestamp
- fix: LogRequest.request is an array
- refactor: change spacing in comments
- docs: add some README documentation

TESTING=manual

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,185 @@
...
+  log_to_promoted_event_api(log_request)
+end
+```
+
+## Example to run the client
+
+1. Clone the repo on your local machine
+2. `cd promoted-ruby-client`
+3. `bundle`
+4. `irb -Ilib -rpromoted/ruby/client`
```
Comment (dev-ham) [inline README.md:31]: **Note**: These instructions were for us to test it. After we publish this. We have to update these.

Comment (danbosnichill) [reply README.md:31]: Makes sense.  The README should have the external customer use.  Do you want these in a different markdown file for development?

File: README.md [hunk 2]
```diff
@@ -1 +1,189 @@
...
+```
+
+## Other input syntaxes
+
+The client should also work if Hash rocket too.
```
Comment (andrewdyates) [inline README.md:101]: rocket?

File: lib/promoted/ruby/client/options.rb [hunk 3]
```diff
@@ -161,17 +150,15 @@ def request_id
 
         def log_request_params
           {
-            platform_id: platform_id,
             user_info: user_info,
             timing: timing,
-            request: request,
+            request: [request],
```
Comment (andrewdyates) [inline lib/promoted/ruby/client/options.rb:155]: why [ ] ?

**Review by andrewdyates — APPROVED** [review]

File: lib/promoted/ruby/client/options.rb (hunk 3 above)
Comment (danbosnichill) [reply lib/promoted/ruby/client/options.rb:155]: `LogRequest.request` is a list field (annoying plural proto naming issue).

File: README.md (hunk 2 above)
Comment (danbosnichill) [reply README.md:101]: Hash Rocket syntax `=>`.  There appears to be a war on which syntax to use.

**Pull request merged**